api/src/wfl/server.clj

```diff
@@ -98,9 +116,10 @@
       (isCancelled [_] false)
       (isDone [_] (.isStopped server)))))
```

Comment:
The start-workload-manager docstring is in the "wrong" place. Intentional?

Reply:
Thanks!